docs/questions/start.md

```diff
@@ -12,7 +12,7 @@ Questions are queries plus their visualization. You can ask questions using Meta
 
 An introduction to Metabase's graphical query builder.
 
-### [Custom expressions](./query-builder/expressions.md)
+### [Custom expressions](./query-builder/expressions-list.md)
```

Comment:
@SethMJarvis Thanks for the PR. I think we should actually leave this link, and change the link below to point to the list page.